domain/model/service/modelservice.go

```diff
@@ -157,3 +212,23 @@ func (s *ModelService) GetStatus(ctx context.Context) (model.StatusInfo, error)
 		Since:  now,
 	}, nil
 }
+
+// GetEnvironProviderVersion retrieves the version of the environment provider associated with the model.
+//
+// Errors:
+// - [modelerrors.NotFound]: Returned if the model does not exist.
+func (s *ModelService) GetEnvironProviderVersion(ctx context.Context) (int, error) {
```

Comment:
Can we rename this to `GetEnvironVersion` ?

Reply:
Sure